### PR TITLE
Documented debug_traceCall

### PR DESCRIPTION
**Pull request by ATREAY**
## Description
This pull request includes information about the **debug_traceCall** method.

## Issue
fixes: #1417

### CODE REVIEW

File: docs/public-networks/reference/api/index.md [hunk 1]
```diff
@@ -1741,6 +1741,77 @@ curl -X POST --data '{"jsonrpc":"2.0","method":"debug_traceBlockByNumber","param
...
+      ]
+    }
+  ]
+}
+```
+<!--/tabs-->
+
+### `debug_traceCall`
+
+Returns a detailed trace of the executed opcodes for a specific transaction.
```
Comment (Gabriel-Trintinalia) [inline docs/public-networks/reference/api/index.md:1774]:
```suggestion
Performs an eth_call within the execution environment of a given block, using the final state of its parent block as the base, and provides a detailed trace of the executed opcodes.
```

File: docs/public-networks/reference/api/index.md [hunk 2]
```diff
@@ -1741,6 +1741,77 @@ curl -X POST --data '{"jsonrpc":"2.0","method":"debug_traceBlockByNumber","param
...
+
+#### Parameters
+
+- `requestContext`: _object_ - JSON-RPC request context with essential fields like `from`, `to`, `gas`, `gasPrice`, `value`, `data`, `nonce`.
```
Comment (Gabriel-Trintinalia) [inline docs/public-networks/reference/api/index.md:1778]:
```suggestion
`call`: _object_ - [transaction call object](objects.md#transaction-call-object)

`blockNumber` or `blockHash`: _string_ - hexadecimal or decimal integer representing a block number, block hash, or one of the string tags `latest`, `earliest`, `pending`, `finalized`, or `safe`, as described in [block parameter](../../how-to/use-besu-api/json-rpc.md#block-parameter)
```

**Review by Gabriel-Trintinalia — CHANGES_REQUESTED** [review]
Some wording changes:

**Review by Gabriel-Trintinalia — COMMENTED** [review]
Hi @ATREAY, lgtm, Could you please add this method to the trace object description in objects.md

**Comment by ATREAY** [comment]
Hi @Gabriel-Trintinalia, I added the debug_traceCall method to the objects.md file. Can you take a look when you have a moment?

**Review by Gabriel-Trintinalia — APPROVED** [review]
lgtm

File: docs/public-networks/reference/api/index.md [hunk 3]
```diff
@@ -1741,6 +1741,78 @@ curl -X POST --data '{"jsonrpc":"2.0","method":"debug_traceBlockByNumber","param
...
+      ]
+    }
+  ]
+}
+```
+<!--/tabs-->
+
+### `debug_traceCall`
+
+Performs an eth_call within the execution environment of a given block, using the final state of its parent block as the base, and provides a detailed trace of the executed opcodes.
```
Comment (alexandratran) [inline docs/public-networks/reference/api/index.md:1774]:
```suggestion
Performs an [`eth_call`](#eth_call) within the execution environment of a given block, using the final state of its parent block as the base, and provides a detailed trace of the executed opcodes.
```

**Review by alexandratran — APPROVED** [review]

**Pull request merged**